src/core/lib/surface/call.cc

```diff
@@ -1259,8 +1259,8 @@ static void post_batch_completion(batch_control* bctl) {
   if (bctl->completion_data.notify_tag.is_closure) {
     /* unrefs bctl->error */
     bctl->call = nullptr;
-    GRPC_CLOSURE_RUN((grpc_closure*)bctl->completion_data.notify_tag.tag,
-                     error);
+    GRPC_CLOSURE_SCHED((grpc_closure*)bctl->completion_data.notify_tag.tag,
```

Comment:
This change seems unrelated to the rest of this PR, since there's no combiner here.  What's the reason for this?

Reply:
There are usecases where this closure is to be run inside in a combiner. As you point out, we are not in a combiner here, so this needed to be changed for correctness.

Reply:
Oh right -- this happens from the grpclb code.  I forgot about that.

Might be worth adding a comment about why this uses sched instead of run.

Reply:
Done!